ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PauseResumeIntegrationTest.java

```diff
@@ -197,8 +202,7 @@ public void pausedQueriesShouldBePausedOnRestart()  {
     // When:
     String queryId = ((Queries) RestIntegrationTestUtil.makeKsqlRequest(REST_APP, "SHOW "
```

Comment:
I think I've sorted out with some refactoring.

Reply:
Sorry, what was the refactoring that was done here? It looks like the test still relies on this line returning the query ID for the query that corresponds to `supplier` rather than `supplier2`.

Reply:
Ok, here's try two.  I'm getting the queryId immediately after creating only that first query.  That should skip over any ordering in the return of `show queries`.

Reply:
Thanks, this makes sense. Can you apply the same update to the other test in this file too?